### ISSUE TITLE
Broken formatting when switching editor from plaintext to HTML

### ISSUE THREAD

**Issue by Player701** [roundcube/roundcubemail#8215]
I have a set up a simple signature in my identity settings, and I have "Compose HTML messages" set to "on forward or reply to HTML message". So by default, when composing a new message, the editor opens in plaintext mode:

![image](https://user-images.githubusercontent.com/9046118/135722283-46e93eb9-e32c-4fca-af21-abfac26768e8.png)

The problem is, if I want to switch to HTML mode, the editor does not honor the default font settings and also disables line spacing (it appears it doesn't add ```<p>``` tags at all).

![image](https://user-images.githubusercontent.com/9046118/135722451-7115d538-d959-4534-9710-863f22a1e0ab.png)

While the font can be changed in the editor (although it can sometimes reset back to "monospace" for unknown reasons), there seems to be no way to fix the latter issue except by editing the message's source code and adding a ```<p>``` tag manually. This is very inconvenient and should probably be fixed somehow.

This issue does not happen if the editor first opens in HTML mode (in my case, when replying to an HTML message or forwarding one). It also doesn't happen if the signature is empty.

![image](https://user-images.githubusercontent.com/9046118/135722402-9424d894-43f6-4428-b9f8-905b1c57bb6f.png)

**Comment by alecpl** [comment]
Yes, our text to html conversion is not perfect. It outputs `<div>`s not `<p>`s. The font issue might be related to that, but might be a separate issue.

**Comment by Player701** [comment]
I've worked around this for now by setting "Compose HTML messages" to "always, except when replying to plain text". This way, the editor always starts in HTML mode, so formatting isn't broken if I want to compose an HTML message - but if I don't then I have to switch to plaintext manually. At least this doesn't require fiddling with the HTML source code of the message.

**Comment by federicoiosue** [comment]
Still not working today on 1.7.4 unluckly.

Any change this to be fixed?
